### PR TITLE
CMS-836 Admins should be able to access all pages in any branch

### PR DESCRIPTION
**Pull request by kowalcj0**
This [ticket](https://uktrade.atlassian.net/browse/CMS-836)

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -214,7 +233,7 @@ def setup_pages_group_permission_user(
 
 
 class BranchEditorFactory:
```
Comment (sdonk) [inline conftest.py]: Same as the other PR. Please keep fixtures only in this file

File: conftest.py [hunk 2]
```diff
@@ -245,3 +283,103 @@ def branch_editor_factory(root_page):
...
+        moderator_is_staff=False
+):
+    """Returns a listing page with a child page, user group, users & clients.
+
+    For Wagtail permission model check:
+    http://docs.wagtail.io/en/v2.0/topics/permissions.html#page-permissions
+    """
+    # ensure that only permissions supported by Wagtail are used
+    available_permissions = [p for p, _, _ in PAGE_PERMISSION_TYPES]
+    assert not (set(editor_permissions) - set(available_permissions))
```
Comment (sdonk) [inline conftest.py]: `ordered` and `==`?

File: conftest.py (hunk 1 above)
Comment (kowalcj0) [reply conftest.py]: all 3 PRs are temporarily "daisy chained". Once this is addressed in #275 then it will be fixed here

File: conftest.py (hunk 2 above)
Comment (kowalcj0) [reply conftest.py]: same answer as in #275 
this will produce a nice assertion error upon failure

**Pull request merged**